docs-src/python/how-to-send-a-signal-from-a-client-in-python.md

```diff
@@ -0,0 +1,16 @@
+---
+id: how-to-send-a-signal-from-a-client-in-python
+title: How to send a Signal from a Client in Python
+sidebar_label: Send a Signal from a Client
+description: Send a Signal from a Client
```

Comment:
To send a Signal to a Workflow Execution from Client code, use the `signal()` method on the Workflow handle.